src/checkers.js

```diff
@@ -413,6 +415,24 @@ function getCheckers(disabled) {
     }, {type}, disabled);
   }
 
+  function lessThanCheckGetter(min) {
+    const type = 'Number';
```

Comment:
For us, the `type` is used in the error message displayed to the user. So perhaps a more descriptive type here would be ``const type = `lessThan[${min}]`;`` or perhaps ``const type = `< ${min}`;``. That way the output might read something like: `` `value` at `Argument 1` must be `lessThan[3]` `` or `` `value` at `Argument 1` must be `< 3` ``